### ISSUE TITLE
Errors on Windows7 with Python3.x installed

### ISSUE THREAD

**Issue by micck** [facelessuser/ScopeHunter#21]
Hello Isaac,

after a 2 year abstinence of using Sublime (or any other Text Editor for that matter), I set up [Sublime Text 3](http://www.sublimetext.com/3dev) (build 3083 currently) on my systems again.  And of course started to install my favourite packages on my new Linux box, or update them on my other systems (two Windows 7 desktops and one MacBook Pro).  Anyway, let's skip to the _issue_ part...

Because of the plugin [WakaTime](https://packagecontrol.io/packages/WakaTime), I needed to install [Python](https://www.python.org/) on my Windows machines.  Out of curiosity, I installed _Python 3.x_ on one and _Python 2.x_ on the other Windows machine.

I got some very strange errors with _ScopeHunter_, on both machines, but **only** in _Sublime Text 3_--in _Sublime Text 2_ (yeah, I've installed both on all my machine for testing purposes) it works without any problems!

I removed all packages, settings, cache files/directories & [Package Pontrol](https://packagecontrol.io/installation) on both machines and started to install my packages one-by-one (always starting with a theme and _ScopeHunter_ as second).  That helped with the machine with _Python 2.x_.

On the machine with _Python 3.x_, I get the following error output, after enabling _ScopeHunter_:

```
reloading Packages/User/Preferences.sublime-settings
ignored packages updated to: ["Vintage"]
reloading plugin ScopeHunter.scope_hunter
ScopeHunter: Theme parsing failed!  Ingoring theme related info.
Traceback (most recent call last):
    File "scope_hunter in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 739, in init_color_scheme
    File "lib.color_scheme_matcher in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 49, in __init__
    File "./plistlib.py", line 104, in readPlistFromBytes
    File "./plistlib.py", line 76, in readPlist
    File "./plistlib.py", line 378, in parse xml.parsers.expat.ExpatError: not well-formed (invalid token): line 191, column 18

Traceback (most recent call last):
    File "%PROGRAMFILES%\Sublime Text 3\sublime_plugin.py", line 116, in reload_plugin
    m.plugin_loaded()
    File "scope_hunter in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 783, in plugin_loaded
    File "scope_hunter in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 765, in init_plugin
    File "scope_hunter in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 686, in __init__
    File "scope_hunter in %APPDATA%\Sublime Text 3\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 725, in setup
AttributeError: 'NoneType' object has no attribute 'is_dark_theme'
reloading Packages/ScopeHunter/scope_hunter.sublime-settings
reloading plugin ScopeHunter.scope_hunter_notify
```

As soon as I click someone in a _Sublime Text_ view with a file loaded, I get the following error message:

```
Traceback (most recent call last):
    File "%PROGRAMFILES%\Sublime Text 3\sublime_plugin.py", line 288, in on_selection_modified
    callback.on_selection_modified(v)
    File "scope_hunter in %APPDATA%\Sublime Text 3\Installed Packages\ScopeHunter.sublime-package", line 622, in on_selection_modified
AttributeError: 'NoneType' object has no attribute 'is_enabled'
```

If I open the command palette, and type in `scope` I only get the menu entry `Scope Hunter: Toggle Instant Scoper`, the entry `Scope Hunter: Show Scope Under Cursor(s)` is missing.

There is no difference if I've some entries in the **scope_hunter.sublime-settings** or not.

Here is my **Package Control.sublime-settings**:
```xml
{
    "in_process_packages":
    [
    ],
    "installed_packages":
    [
        "Package Control",
        "ScopeHunter",
        "Theme - Cola"
    ]
}
```

Hope that helps.

Tomorrow, I'll de-install _Python 3.x_ and install _Python 2.x_ as well upgrade _Sublime Text 3_ to build 3094 in order to gather more information.

Take care,
Mick

**Comment by facelessuser** [comment]
Looks like it can't parse your color scheme.  The  plistlib included with python looks to be choking.  I can't change plistlib.

Even if sublime can parse it, it looks like python can't.  Which color scheme are you using? I can probably identify what is wrong with the scheme so it can be patched by the maintainer.

**Comment by micck** [comment]
I'm using my own color scheme--currently rewriting it from scratch--, but wait maybe it is the theme. Thanks for pointing that out. Will check later.

**Comment by micck** [comment]
:flushed: I'm ashamed. :blush: 2 years ago, I opened a issue and the reason for it was the way I do comments.  :sob: I will now vanish from the face of the :earth_africa: and only reappear :dizzy:, if I know the syntax of [plist](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/PropertyLists/UnderstandXMLPlist/UnderstandXMLPlist.html) by :heart: 

**tl;dr:** Yeah, it's my color scheme--_again_.

**Comment by facelessuser** [comment]
No worries :smile:.  Anytime I see something like this:

 ```
File "./plistlib.py", line 378, in parse xml.parsers.expat.ExpatError: not well-formed (invalid token): line 191, column 18
```

I know its the scheme.  Glad you figured it out though.

**Comment by micck** [comment]
Yeah, I should have read the output more thoroughly, than I would have know :( thanks again!
